src/screens/QuestionScreen/QuestionScreen.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { View, Text, Image, TouchableOpacity, SafeAreaView, TextInput, Alert, Linking } from 'react-native';
+import { firebase } from '../../config/config';
+import { LOGO } from '../../config/styles.js';
+import styles from './styles';
+import moment from 'moment';
+import Icon from 'react-native-vector-icons/Ionicons';
+import ImagePicker from 'react-native-image-crop-picker';
+import * as Progress from 'react-native-progress';
+
+class QuestionScreen extends  React.Component {
+    state = {QueryInput : '', setimage : null, setDisplayImage: null, transferred: 0};
+
+    takephotofrommlib = () => {
+        const {setimage} = this.state
+        this.setState({transferred: 0})
+        ImagePicker.openPicker({
+          width: 300,
+          height: 300,
+          cropping: true,
+        }).then((image) => {
+          const imageUri = Platform.OS === 'ios' ? image.path : image.path;
+          this.setState({setimage: imageUri});
+        }).catch("Unknown Error Occured")
+      };
+
+      uploadImage = async () => {
+        const {setimage} = this.state
+        if(setimage == null){
+          return
+        }
+          const uri = setimage;
+          const childPath = `post/${firebase.auth().currentUser.uid}/${Math.random().toString(36)}`;
```

Comment:
Let's change this URI to post/uid/<unique id of the post>/random_string. This way I think the files be more organized. As a suggestion, let's have a variable listed in firebase that hold a integer value which will be the next post id. I think this can be declared in user profile data section. Value should ne updated once new post is added.

Reply:
Okay! I'll change it, but I have no idea about the use of '//'. What is the use of it?

Reply:
I didn't understand this approach.  Do you want me to add a new Integer field in User profile data (with Full name, email, interest, country) which will contain the post id (next?)? What it will be used for?

Reply:
Sorry, I missed what I meant here. I have a question, when we give answerers, are they also posted into the same object that is created in the post/uid URI?